### PR TITLE
DOP-918: Build limited legacy sites

### PR DESCRIPTION
**Pull request by sophstad**
[[JIRA](https://jira.mongodb.org/browse/DOP-918)] [[Manual v2.6 Staging](https://docs-mongodbcom-staging.corp.mongodb.com/sophstad/v2.6-eol/index.html)] [[K8s Operator v1.5 Staging](https://docs-mongodbcom-staging.corp.mongodb.com/kubernetes-operator/sophstad/v1.5-eol/index.html)] [[Ops Manager v4.0 Staging](https://docs-opsmanager-staging.mongodb.com/sophstad/v4.0/index.html)] [[Non-EOL K8s Operator](https://docs-mongodbcom-staging.corp.mongodb.com/kubernetes-operator/sophstad/master/index.html)] For deprecated versions:
- Remove navbar
- Deactivate version dropdown
- Add link to interstitial legacy page
- Add deprecation warning banner to all pages with optional link to upgrade documentation

To configure a branch for deprecation, in its `conf.py` file, within `html_theme_options`, ensure that `eol` is `True` and add a `eol_upgrade_url` link if desired.

### CODE REVIEW

**Comment by schmalliso** [comment]
Is there a corresponding theme_eol thing we're handling for nextgen repos that are no longer supported (if/when such a thing comes up)?

File: themes/mongodb/layout.html [hunk 1]
```diff
@@ -204,7 +217,23 @@
 
     <span id="showNav" class="showNav">Navigation</span>
 
-      {% block alertbar %}{% endblock %}
+      {% block alertbar %}
+        {%- if theme_eol %}
+            <div class="alert alert-warning">
+              <span class="alert-message">This version of the documentation is archived and no longer supported.{% if theme_eol_upgrade_url %} Learn more about <a href="{{ theme_eol_upgrade_url }}">upgrading</a> your version of {{ theme_sitename }}.{% endif %}</span>
+            </div>
```
Comment (schmalliso) [inline themes/mongodb/layout.html:224]: should this span and div be before the `{% endif %}`?

**Comment by sophstad** [comment]
> Is there a corresponding theme_eol thing we're handling for nextgen repos that are no longer supported (if/when such a thing comes up)?

@schmalliso This is definitely a TODO! Probably good to start thinking about sooner than later, and also worth considering if/how we want deprecated next-gen sites to be different from those built on legacy tooling.

**Comment by eshwer** [comment]
Worth asking @allisonmui9's thoughts on look/feel for nextgen legacy properties

File: themes/mongodb/layout.html (hunk 1 above)
Comment (schmalliso) [reply themes/mongodb/layout.html:224]: READING IS HARD. NEVER MIND.

**Comment by jdestefano-mongo** [comment]
@sophstad LGTM mod one request and one question:

For the manual, the message should read: "To learn how to upgrade your version of MongoDB, refer..."
manual-eol

Can writers control the message or is that set in the tools theme?

**Review by jdestefano-mongo — APPROVED** [review]
lgtm, thanks for the manual's fix!

**Pull request merged**